mandatory/1-fix-functions.js

```diff
@@ -10,20 +10,20 @@
 
 */
 
-function mood() {
+function mood(value) {
   let isHappy = true;
 
-  if (isHappy) {
+  if (isHappy === value) {
```

Comment:
You can remove the `isHappy` declaration here, as `value` is already a boolean; so you can say:
```
if (value) {
   return 'I am happy
}
return 'I am not happy'
```